payment-cards/pci-compliance/front-end.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: PCI on the Front-End
```

Comment:
Would you want to re-name this to Client-Side (vs Server-Side), I think that would reflect the terminology used by Braintree, Stripe et. al better